plasma/client/client.py

```diff
@@ -48,7 +49,14 @@ def submit_block(self, block):
     def withdraw(self, blknum, txindex, oindex, tx, proof, sigs):
         utxo_pos = encode_utxo_id(blknum, txindex, oindex)
         encoded_transaction = rlp.encode(tx, UnsignedTransaction)
-        self.root_chain.startExit(utxo_pos, encoded_transaction, proof, sigs, transact={'from': '0x' + tx.newowner1.hex()})
+
+        _from = ''
```

Comment:
Maybe we could do something like:

```
owner = tx.newowner1 if oindex == 0 else tx.newowner2
from = address.to_checksum_address('0x' + owner.hex())
```

Reply:
All right

the `from` word is one of the python keywords, i'll use `owner_addr` instead.